src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
Now this view has skipped to the multispeed coil function (so no more changes to CalcDXHeatingCoil). I have to go to the code to see what's next. This would make me ask why change the power calculation in Multispeed coil and not single-speed coil?